Sources/MapboxNavigation/MultiplexedSpeechSynthesizer.swift

```diff
@@ -40,6 +40,11 @@ open class MultiplexedSpeechSynthesizer: SpeechSynthesizing {
         speechSynthesizers.forEach { $0.locale = locale }
     }
     
+    /// Controls if this speech synthesizer is allowed to manage the shared `AVAudioSession`.
+    /// Set this field to `false` if you want to manage the session yourself, for example if your app has background music.
+    /// Default value is`true`.
+    public var managesAudioSession = true
```

Comment:
This has to be a computed property which will forward `newValue` to underlying speech synthesizers.

Reply:
Good idea, thanks